docs/guide/src/pcli/install.md

```diff
@@ -1,79 +1,24 @@
-# Installing `pcli`
+# Installing pcli
 
-### Installing the Rust toolchain
+Download prebuilt binaries from the [Penumbra releases page on Github](https://github.com/penumbra-zone/penumbra/releases).
+Make sure to use the most recent version available, as the version of `pcli` must
+match the software currently running on the network.
 
-This requires that you install a recent (>= 1.73) stable version
-of the Rust compiler, installation instructions for which you can find
-[here](https://www.rust-lang.org/learn/get-started). Don't forget to reload your shell so that
-`cargo` is available in your `\$PATH`!
+Make sure choose the correct platform for your machine. After downloading the `.tar.xz` file,
+extract it, and copy its contents to your `$PATH`. For example:
 
-You can verify the rust compiler version by running `rustc --version` which should indicate version 1.73 or later.
-
-`pcli` requires `rustfmt` as part of the build process --- depending on your
-OS/install method for Rust, you may have to install that separately.
-
-### Installing build prerequisites
-
-#### Linux
-
-You may need to install some additional packages in order to build `pcli`,
-depending on your distribution. For a bare-bones Ubuntu installation, you can
-run:
-
-```bash
-sudo apt-get install build-essential pkg-config libssl-dev clang git-lfs
 ```
+curl -O -L https://github.com/penumbra-zone/penumbra/releases/download/{{ #include ../penumbra_version.md }}/pcli-x86_64-unknown-linux-gnu.tar.xz
+unxz pcli-x86_64-unknown-linux-gnu.tar.xz
+tar -xf pcli-x86_64-unknown-linux-gnu.tar
+sudo mv pcli-x86_64-unknown-linux-gnu/pcli /usr/local/bin/
```

Comment:
Nit: should we use a macOS arch for the pcli example? I think saying explicitly: "Make sure choose the correct platform for your machine" is good enough, but open to feedback.